pkgs/development/tools/haskell/dconf2nix/default.nix

```diff
@@ -0,0 +1,25 @@
+{ mkDerivation, base, containers, optparse-applicative, parsec
+, lib, text
+}:
+
+mkDerivation rec {
+  pname   = "dconf2nix";
+  version = "0.0.5";
+
+  src = builtins.fetchurl {
```

Comment:
GitHub-generated archives can change hashes. Use `fetchFromGitHub`, which hashes the archive’s content.